README.md

```diff
@@ -73,7 +73,8 @@ To deploy an existing model on your machine, you can either 1) specify the name
 ### Option 1: use a model from the Viam registry
 
 You can download an ML model from Viam's registry. Head to
-https://app.viam.com/registry?type=ML+Model to see the available models. Once you've added this
+https://app.viam.com/registry?type=ML+Model to see the available models. Choose a model that uses either
+the TensorFlow or PyTorch model framework; this service does not support TensorFlow Lite. Once you've added this
```

Comment:
Rather than having the list of supported models in two places, maybe link back to once place that has it.